src/sections/offering/insured-amount.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react'
+import styled from 'react-emotion'
+import { colors } from '@hedviginsurance/brand'
+import { Header } from './price-info'
+
+interface Props {
+  title: string
+  info: string
+}
+export const Container = styled('div')({
+  width: '100%',
+  backgroundColor: colors.OFF_WHITE,
+})
+
+export const InnerContainer = styled('div')({
+  display: 'flex',
+  alignItems: 'center',
+  flexDirection: 'column',
+})
+
+export const Card = styled('div')({
```

Comment:
If we use these in a lot of places we should prolly move them to ie `/src/components`